docs/source/science/NISAR/OPERA Sentinel-1.ipynb

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
<p>Can you insert a map of the granule bounding boxes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>